cli/consts.go

```diff
@@ -17,7 +17,7 @@ var Dev bool = false
 const DefaultTTL uint = 1
 
 // DefaultTimeRemaining for new key requests in minutes
-const DefaultTimeRemaining uint = 60
+const DefaultTimeRemaining uint = 5
```

Comment:
This was dropped to 5 minutes because the default TTL is 1 hour, which means that for any user that doesn't specify `--time-remaining` or `--ttl`, the behaviour of `keyconjurer get` will be to *always* fetch new credentials even if there are perfectly legitimate ones in the environment